distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -710,7 +710,7 @@ type Array
 
              [0, 1, 2].to_array . reduce (+)
     reduce : (Any -> Any -> Any) -> Any -> Any
-    reduce self function ~if_empty=(Error.throw Empty_Error) = Vector.reduce self function if_empty
+    reduce self function ~if_empty=(Error.throw (Empty_Error.Error Vector)) = Vector.reduce self function if_empty
```

Comment:
Shouldn't this be
```suggestion
    reduce self function ~if_empty=(Error.throw (Empty_Error.Error Array)) = Vector.reduce self function if_empty
```

given that we are operating with an Array here??

I know they are pretty much interchangeable but they still are separate types.

(Sorry I just noticed it after getting a merge conflict with #8867)

Reply:
Yes. Good spot :) I will change that.

Reply:
https://github.com/enso-org/enso/pull/9019